.cargo/config

```diff
@@ -1,2 +1,13 @@
 [alias]
 regen-protobufs = ["run", "--bin", "protobuf-gen", "tools/protobuf_files.toml"]
+dev-install = ["install", "asdev"]
+all_tests = ["asdev", "test"]
+rust_tests = ["asdev", "test_rust"]
+verify_env = ["asdev", "verify_env"]
+smoke_android = ["asdev", "smoke_android"]
+smoke_fenix = ["asdev", "smoke_fenix"]
+smoke_ios = ["asdev", "smoke_ios"]
+tag_minor = ["asdev", "tag_minor"]
+prepare_release = ["asdev", "prepare_release"]
+lint_bash = ["asdev", "lint_bash"]
+update_pr = ["asdev", "cargo_update"]
```

Comment:
let's remove the above for now and fix these up a bit. I created an issue for the `prepare_release` script https://github.com/a-s-dev/asdev/issues/15

Reply:
Done, got rid of those for now, until I clean up the tool more